backend/tests/integration/submissions/test_submissions_views.py

```diff
@@ -1727,8 +1733,9 @@ def test_confirmation_email_not_sent_when_user_is_cataloger(
 
 
 @patch("inspirehep.submissions.views.send_conference_confirmation_email")
+@patch("inspirehep.submissions.views.async_create_ticket_with_template")
```

Comment:
You don't need this `patch` it has already :)

Reply:
hmm, okay but it then has the same error as the ones that didn't have the ```patch```

Reply:
Ok I see, then we have to put this patch as well.

Reply:
yep, then just need to figure out a way to 'fix' the ```assertion error``` it creates.